unit-tests/live/options/test-rgb-options-metadata-consistency.py

```diff
@@ -81,15 +81,19 @@ def check_option_and_metadata_values(option, metadata, value_to_set, frame):
                 # the following if statement is needed because of some bug in FW - see DSO-17221
                 # to be removed after this bug is solved
                 if option == rs.option.white_balance:
+                    log.d("iteration", iteration, "setting explicitly enable_auto_white_balance to OFF")
                     color_sensor.set_option(rs.option.enable_auto_white_balance, 0)
+                    log.d("enable_auto_white_balance new value we read back is:", color_sensor.get_option(rs.option.enable_auto_white_balance))
```

Comment:
Here we are not waiting. Are you expecting an updated value?

Reply:
Yes, I will add a check for that

Reply:
Done